### DISCUSSION TITLE
How do I remove the homepage?

### DISCUSSION THREAD

**Discussion by annalhq** [fuma-nama/fumadocs#418] [Q&A] [answered]
I don't want to have that homepage (page.tsx) as my default root page. Instead I want my app to have /docs setup as the homepage by default, just like Nextra has. 

Any help would be appreciated thanks!

**Comment by fuma-nama** [comment, marked as answer]
You can change the routing structure. As we are using the normal Next.js Routing, you can simply move the folders/files, and modify the base URL option in `source.ts` to change the generated page URLs.

In the repository, there's an I18n example that uses the same routing structure. You may take it as a reference.
